### DISCUSSION TITLE
screw holes

### DISCUSSION THREAD

**Discussion by ja2ui0** [joshreve/dactyl-keyboard#17] [General]
What is the parameter that describes the diameter of the holes for the heat set inserts at the bottom of the case (not the plate)? I just noticed that the holes in all my test prints are much too big for my inserts, and I measured said holes at 5.30945mm diameter according to FreeCAD. My M3 inserts want a hole of not much greater than 4mm - they are swimming in the ones being printed.

[These are the inserts I got](https://www.amazon.com/gp/product/B08BJD3W4X).

**Comment by ja2ui0** [comment]
Nevermind my question, it looks like it's in `dactyl_manuform.py` hardcoded by:

```
screw_insert_height = 3.8
screw_insert_bottom_radius = 5.31 / 2
screw_insert_top_radius = 5.1 / 2
```

I'm thinking that those values should be parameterized and represented by a single value, need to experiment with it.

**Reply by joshreve** [reply]
That is a good request.  I can move them to the config file.  I intended to get all the relevant parameters when I shifted to the json / configuration setup, but I must have missed them.

**Reply by freznel10** [reply]
Using cadquery, my test prints take M3 inserts well. The problem is that when I cut the acrylic for the plate.. the default 2mm screw hole  size in the config should be change to match the default inserts.
